spec/pagerduty/legacy_spec.rb

```diff
@@ -2,21 +2,27 @@
 
 require "spec_helper"
 
-describe Pagerduty do
+RSpec.describe Pagerduty do
```

Comment:
Here we prove we still support the old gem 2.0 interface by running the original tests. They all still pass bar one.